MacCopyPaste/MacCopyPaste/Classes/ImageDocument.cs

```diff
@@ -90,12 +90,15 @@ public ImageDocument ()
 			if (ok) {
 				// Read the image off of the pasteboard
 				NSObject [] objectsToPaste = pasteboard.ReadObjectsForClasses (classArray2, null);
-				ImageInfo info = (ImageInfo)objectsToPaste[0];
+                	if (objectsToPaste.Length > 0)
```

Comment:
Tab indent is wrong here.

Reply:
Also, info doesn't appear be used?

Reply:
Formatting updated. info is not used anywhere but objectsToPaste is type-casted which was crashing earlier.